napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -672,11 +674,13 @@ def _add_to_installed(distname, enabled, npe_version=1):
                 npe_version=npe_version,
             )
 
-        for manifest in _npe2.iter_manifests():
+        pm2 = PluginManager.instance()
+        for manifest in pm2.iter_manifests():
             distname = normalized_name(manifest.name or '')
             if distname in self.already_installed or distname == 'napari':
                 continue
-            _add_to_installed(distname, True, npe_version=2)
+            enabled = not pm2.is_disabled(manifest.name)
+            _add_to_installed(distname, enabled, npe_version=2)
```

Comment:
The `npe_version` parameter bothers me but I don't know what to do about it. It's not really what this PR is about, but I wonder if there's a better name: `npe_backend_id`?

Reply:
maybe just `npe_id`

Reply:
it's gonna be gone anyway after #4015